aiger_analysis/cadet.py

```diff
@@ -82,9 +85,11 @@ def is_true_QBF(e, quantifiers):
             assert quantifiers[0][0] is 'e'
             return is_satisfiable(aig)
     elif len(quantifiers) is 2:  # 2QBF
-        if quantifiers[0][0] is 'e':
+        true_return_code = 10
```

Comment:
might be easier to read if we make an enum for the error codes:

```python
from enum import Enum

class CadetCodes(Enum):
     MEANINGFUL_NAME = 10
     ALSO_MEANINGFUL_NAME = 2
```

And then `true_return_code = CadetCodes.Meaningful_Name`

Reply:
Yeah, probably good for documentation purposes. But makes the code more cluttered.